docs/chains/lava-chain/lava.md

```diff
@@ -42,7 +42,11 @@ https://raw.githubusercontent.com/lavanet/lava/main/cookbook/specs/spec_add_lava
 ```bash
 lavad q spec show-chain-info LAV1
 ```
+#### ⌨️ Command for Mainnet:
 
+```bash
+lavad q spec show-chain-info lava-mainnet-1
```

Comment:
It doesn't work, because you need to set chain name, not chain-id.
```
lavad query spec show-chain-info [chain-name] [flags]
```

Reply:
Yeah i think the same LAV1